README.md

```diff
@@ -42,3 +42,10 @@ For example, if the instance connection name is `foo:bar:baz`, the JDBC URL
 would be 
 `jdbc:mysql://google/mydb?cloudSqlInstance=foo:bar:baz&socketFactory=com.google.cloud.sql.mysql.SocketFactory`
 
+## Credentials
```

Comment:
Also would be helpful if it was possible to specify client_email and private_key (as defined in JSON file) as the URL parameters. This would allow for easy app parameterisation and avoid dealing with the key files.